python/tvm/relay/interpreter.py

```diff
@@ -0,0 +1,130 @@
+#pylint: disable=no-else-return
+"""An interface to the Realy interpreter."""
+from __future__ import absolute_import
+import numpy as np
+from .. import register_func, nd
+from .base import NodeBase, register_relay_node
+from . import _make
+from . import _interpreter
+from . import ir_pass
+from .expr import Call, Constant, GlobalVar
+from . import const
+
+
+class Value(NodeBase):
+    """Base class of all values.
+    """
+
+    @staticmethod
+    @register_func("relay.from_scalar")
+    def from_scalar(i, dtype=None):
+        """Convert a Python scalar to a Realy scalar."""
+        if dtype is None:
+            if isinstance(i, int):
+                dtype = 'int32'
+            elif isinstance(i, float):
+                dtype = 'float32'
+            elif isinstance(i, bool):
+                dtype = 'uint8'
```

Comment:
```suggestion
                dtype = 'uint1'
```

Reply:
I'm pretty sure this must be `uint8` since we can't actually allocated `uint1` on CPU. @tqchen ?

Reply:
It should be "bool", https://github.com/dmlc/tvm/pull/1853